tests/client.auth.test.js

```diff
@@ -30,7 +30,8 @@ describe('auth test', function() {
 
   describe('if type is disabled', function() {
     it('subscribe fails and emits err', function(done) {
-      client.on('err', function(message) {
+      client.on('err', function(response) {
```

Comment:
this is bad. We should not change existing behavior. We should send response.getMessage()